akka-actor-typed-tests/src/test/scala/akka/actor/typed/pubsub/LocalPubSubSpec.scala

```diff
@@ -68,6 +68,7 @@ class LocalPubSubSpec extends ScalaTestWithActorTestKit with AnyWordSpecLike wit
           .debug("Topic list updated")
           // both topic instances should have seen the updated list
           .withOccurrences(2)
+          .withCheckExcess(false)
```

Comment:
I don't think it's possible to know that it will be exactly 2 "Topic list updated". It will be at least 2.

Alternatively we could remove the logging asserts altogether because they are not core to what the test is verifying, as far as I can see.

Reply:
I think we could remove it, I think it could be that the log assert was added here before the GetTopicStats message existed, but new the rest is protected against racing by that instead.

Reply:
ok, I'll remove them instead

Reply:
https://github.com/akka/akka/pull/30642/commits/42a81cd32e6e58479e1ad29e01e4a361ec9744f2